core/lib/dal/src/blocks_dal.rs

```diff
@@ -249,7 +249,7 @@ impl BlocksDal<'_, '_> {
             WHERE
                 number = $1
             "#,
-            number.0 as i64
+            i64::from(number.0)
```

Comment:
prob deny it on the [lint level](https://rust-lang.github.io/rust-clippy/master/index.html#/as_conversions) then?